Assets/Scripts/MoveDragon.cs

```diff
@@ -175,19 +174,7 @@ public class MoveDragon : MonoBehaviour
     /** Send a raycast to check if player is grounded and returns true if
      the player is on some sort of ground */
     private bool IsGrounded() {
-        // boxcast not working now
-        // float extraHeight = 0.05f;
-        // bool hitGround = Physics.BoxCast(DragonCollider.bounds.center, DragonTransform.lossyScale, DragonTransform.up * -1, Quaternion.Euler(Vector3.zero), DragonTransform.lossyScale.y + extraHeight);
         bool hitGround = Physics.Raycast(DragonTransform.position, Vector3.down, 0.5f);
-
-        Color rayColor;
-        if (hitGround) {
-            rayColor = Color.green;
```

Comment:
What is the reason for this change? Is this not needed?

Reply:
yes, this is just a test block copied from movePB.cs